<div>
  <h3>Abstract</h3>
  <p>
    Measuring the chemical reactivity of a nucleotide within a sequence can infer the local conformation of a nucleotide most of the time. The simplest interpretation of the local conformation of a nucleotide is by the term paired or non-paired. However, these two states do not fully explain the reactivity of nucleotides in chemical probing experiments.

    Currently, the RNA mapping database (RMDB) contains 131,271 sequences probed under different conditions, which gives us a relatively good data set to learn using machine learning algorithms. In my research project, I attempted to predict and understand the reactivity or non-reactivity of nucleotides of thousand of RNA obtained from RMDB. For this, I used two secondary structure RNA folding algorithms: MCFlashFold from Dr. Fran√ßois Major's laboratory and RNAsubopt from the Vienna package collection. The originality of my project is that it is based on an abstraction of the secondary structure called nucleotide cyclic motif.
    In this thesis, I demonstrate that the majority of the discretized (high / low) reactivities values of nucleotides can be predicted, but not all. The proposed interface gives us the opportunity to quickly and easily compare homolog nucleotides between RNA.

    You will also find that the number of measurable features of a nucleotide exponentially increases when nucleotide neighbors are considered. So, the question is what characteristics should be extracted to accurately predict the reactivity of a nucleotide?

    <h4>Keywords : RNA, NCM, secondary structure, SHAPE, Eterna, RMDB</h4>
</p>
</div>
